### PR TITLE
Minor updates to JSON changes

### PR DESCRIPTION
**Pull request by MaryamZi**
## Purpose
$title.

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md [hunk 1]
```diff
@@ -167,7 +167,7 @@ type Employee record {|
 
 ### Improvements related to JSON compatibility
 
-Three langlib functions were introduced and `toJsonString` is now allowed to have `anydata`.
+Three new langlib methods have been introduced to improve converting to and from JSON. Additionally, the `toJsonString` method can now be called on `anydata` values.
```
Comment (pubudu91) [inline downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md:170]: We don't really use the term "langlib" in docs like these right?

File: downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md [hunk 2]
```diff
@@ -167,7 +167,7 @@ type Employee record {|
...
 
 #### `toJson`
```
Comment (pubudu91) [inline downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md:172]: Also, should we consider making these headings a bit more descriptive? When viewing the generated page, this looks weird with the formatting.

File: downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md (hunk 1 above)
Comment (MaryamZi) [reply downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md:170]: How about something like 

```suggestion
Three new methods have been introduced to the `ballerina/lang.value` module to facilitate converting to and from JSON. Additionally, the `toJsonString` method can now be called on `anydata` values.
```

Comment (pubudu91) [reply downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md:170]: +1

File: downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md (hunk 2 above)
Comment (MaryamZi) [reply downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md:172]: We have a bit more content now. :smile:

Comment (pubudu91) [reply downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md:172]: No no, I meant the heading `toJson` itself.

Comment (MaryamZi) [reply downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md:172]: If we are changing this, the only change I can think of is making it something like
```suggestion
#### The `toJson` method
```

Comment (pubudu91) [reply downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md:172]: Yeah +1. That's what I was thinking about as well

**Pull request merged**